public/css/quiz-rule.css

```diff
@@ -0,0 +1,81 @@
+@import url('https://framer-ui.netlify.app/components/framer.css');
+
+:root{
+    --navbar-bg:#2C2929;
+    --img-text:#f1f2f3;
+    --img-banner:#4F46E5;
+
+}
+
+.navbar-bg{
+    background: var(--navbar-bg);
+}
+
+.nav-zindex{
+    z-index: 9;
+}
```

Comment:
why zindex 9 ?

Reply:
> why zindex 9 ?

Basically, the image container was hiding the navbar component.